spec/helpers/recipes_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the RecipesHelper. For example:
+#
+# describe RecipesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe RecipesHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ]  From the model, we can test the association you did in the database configuration, kindly test the association between the tables. For example: `it { should belong_to(:recipe) }`. Please make sure you have tested all the associations.
- [ ] Implement the tests for the model